keras_retinanet/bin/debug.py

```diff
@@ -154,6 +155,7 @@ def csv_list(string):
     parser.add_argument('-l', '--loop', help='Loop forever, even if the dataset is exhausted.', action='store_true')
     parser.add_argument('--no-resize', help='Disable image resizing.', dest='resize', action='store_false')
     parser.add_argument('--anchors', help='Show positive anchors on the image.', action='store_true')
+    parser.add_argument('--display-name', help='Display image name on the bottom left corner.', dest='name', action='store_true')
```

Comment:
I don't think there's a need for the `dest='name'` part, having the argument as `dest_name` will be fine.

Reply:
hmm not sure if i understood, like this: 
```'--display-name', help='Display image name on the bottom left corner.', action='store_true'```

Reply:
Yeah, so without the `dest`